xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/PanelSheet.xml

```diff
@@ -53,14 +53,21 @@
 #set ($obj = $doc.getObject('Panels.PanelClass'))
 #if ($obj)
   #set($class = $obj.xWikiClass)
-  #displayPanelProperty($obj 'name')
-  #displayPanelProperty($obj 'type')
-  #displayPanelProperty($obj 'category')
-  #displayPanelProperty($obj 'description')
-  #displayPanelProperty($obj 'content')
-  #displayPanelProperty($obj 'async_enabled')
-  #displayPanelProperty($obj 'async_cached')
-  #displayPanelProperty($obj 'async_context')
+  ## Enclose in .xform class div when editing
+  #if ($xcontext.action == 'edit')
+    {{html clean=false}}&lt;div class='xform'&gt;{{/html}}
```

Comment:
You can do it like this or you could have used wiki syntax, something like `(%class='xform'%)((()))`

Also you should have used apostrophe (') for when giving the value for the `clean` parameter.